### PR TITLE
Fixes #9826: Remove unneeded dependencies from our comps.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Test repos generated from these comps changes: http://koji.katello.org/releases/yum/ehelms-test/

File: rel-eng/comps/comps-katello-server-rhel6.xml [hunk 1]
```diff
@@ -11,12 +11,18 @@
...
+       <packagereq type="default">ruby193-rubygem-tire</packagereq>
+       <packagereq type="default">lucene4</packagereq>
+       <packagereq type="default">lucene4-contrib</packagereq>
+       <packagereq type="default">sigar-java</packagereq>
+       <packagereq type="default">sigar</packagereq>
+       <packagereq type="default">snappy-java</packagereq>
+
+       <!-- katello core packages -->
        <packagereq type="default">katello</packagereq>
-       <packagereq type="default">katello-agent</packagereq>
```
Comment (jlsherrill) [inline rel-eng/comps/comps-katello-server-rhel6.xml]: I kinda feel that we shouldn't remove katello-agent until http://projects.theforeman.org/issues/7760 is resolved

Comment (ehelms) [inline rel-eng/comps/comps-katello-server-rhel6.xml]: Shouldn't the client repo being providing this though?

Comment (jlsherrill) [inline rel-eng/comps/comps-katello-server-rhel6.xml]: It should, it just doesn't exist today.  Today people can easily install the normal katello-repos rpm and get the same result, i feel we should provide the same level of functionality before we remove the agent from the repo.

Comment (jlsherrill) [inline rel-eng/comps/comps-katello-server-rhel6.xml]: Also keep in mind that the capulse will need katello-agent, today they can just install this one rpm.  With this change they can't (have to configure the client repo directly).

Comment (ehelms) [inline rel-eng/comps/comps-katello-server-rhel6.xml]: I believe this is now resolved with the creation of the client repo RPM ?

**Comment by ehelms** [comment]
Closing in favor of https://github.com/Katello/katello-packaging/pull/22

**Pull request closed without merging**